### PR TITLE
Remove infra-nodes form node count

### LINKED ISSUE #93: [BUG] ROSA nodes with infra label
**Describe the bug**
ROSA deploys with `worker=` and `infra=` on the "infra" nodes. Our rules only look for `worker=` so client/servers could be scheduled on incorrect nodes.

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- ROSA deploys infra nodes with the worker= label. Adding code to ignore those nodes, and to schedule on nodes without the infra label.

## Related Tickets & Documents

- Related Issue #93 
- Closes #93

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: pkg/k8s/kubernetes.go [hunk 1]
```diff
@@ -78,15 +78,28 @@ func BuildSUT(client *kubernetes.Clientset, s *config.PerfScenarios) error {
 	if err != nil {
 		return err
 	}
-	ncount := len(nodes.Items)
+
+	ncount := 0
+	for _, node := range nodes.Items {
+		if _, ok := node.Labels["node-role.kubernetes.io/infra"]; ok {
```
Comment (vishnuchalla) [inline pkg/k8s/kubernetes.go:84]:
```suggestion
       		if _, ok := node.Labels["node-role.kubernetes.io/infra"]; !ok {
			ncount++
	}
```

Comment (vishnuchalla) [reply pkg/k8s/kubernetes.go:84]: Just a small nit. Thank you!

Comment (jtaleric) [reply pkg/k8s/kubernetes.go:84]: I only want to count nodes with `worker=`  -- and ignore nodes with `infra` or nodes with both `worker` and `infra`.

Comment (vishnuchalla) [reply pkg/k8s/kubernetes.go:84]: IMHO, yes that check should simply ignore every node with `infra` label that covers both cases `infra` or `worker` and `infra`.

**Review by vishnuchalla — APPROVED** [review]
lgtm!

**Pull request merged**